config/config.json

```diff
@@ -134,6 +135,7 @@
                         "package": "./blockchain/implementation/ot-parachain/ot-parachain-service.js",
                         "config": {
                             "networkId": "otp::testnet",
+                            "identityFileName": "otp",
```

Comment:
We are using only Parachain devnet at the moment, but we are calling it parachain testnet in the ot-node, this is not issue at the moment, but in future we will introduce parachain testnet also, so just heads up, probably better not to change this now but later